backend/t4cclient/sessions/files/routes.py

```diff
@@ -0,0 +1,83 @@
+# Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard library:
+import io
+import logging
+import tarfile
+
+import requests
+
+# 3rd party:
+from fastapi import APIRouter, Depends, HTTPException, UploadFile
+from requests.auth import HTTPBasicAuth
+from sqlalchemy.orm import Session
+
+# local:
+from t4cclient.core.authentication.database import check_session_belongs_to_user
+from t4cclient.core.authentication.helper import get_username
+from t4cclient.core.authentication.jwt_bearer import JWTBearer
+from t4cclient.core.database import get_db
+from t4cclient.core.oauth.responses import AUTHENTICATION_RESPONSES
+from t4cclient.sessions.database import get_session_by_id
+from t4cclient.sessions.operators import OPERATOR
+from t4cclient.sessions.schema import FileTree
+
+router = APIRouter()
+log = logging.getLogger(__name__)
+
+
+@router.get("/", response_model=FileTree)
+def get_files(
+    id: str,
+    show_hidden: bool,
+    db: Session = Depends(get_db),
+    token=Depends(JWTBearer()),
+):
+    check_session_belongs_to_user(get_username(token), id, db)
+    session = get_session_by_id(db, id)
+
+    return requests.get(
+        "http://" + session.host + ":8080/api/v1/workspaces/files",
+        params={"show_hidden": True},
```

Comment:
Should `show_hidden` be given as a function parameter? Then, it should be `params={"show_hidden": show_hidden}` and `show_hidden` has to be assigned from the frontend.
Otherwise, line 33 can be deleted or maybe `show_hidden` can be defined in a values.yaml.